### ISSUE TITLE
Use lib prefix for library packages for all architectures

### ISSUE THREAD

**Issue by Conan-Kudo** [OpenMandrivaAssociation/OpenMandrivaAssociation.github.io#54]
Now that we use a combined multi-arch repository for 32-bit and 64-bit libraries for the x86_64 repository, we should also stop mutating the package name based on which libdir it installs into. RPM and DNF are perfectly capable of doing architecture selection properly for multiarch, so we can just have `libfoo1` instead of `libfoo1` and `lib64foo1`.

**Comment by berolinux** [comment]
Agreed, but let's hold off on this until 5.0 -- 4.2 is just about ready, let's not make another big change there.
